### ISSUE TITLE
Highlight color on revision comparison lacks color contrast

### ISSUE THREAD

**Issue by jessemenn** [wagtail/wagtail#10552]
### Issue Summary

In dark mode, the highlight color on revisions for deletions/additions are rather poor and unreadable. I think it's 1.22:1 and 1.35:1 respectively.

### Steps to Reproduce

1. Edit something. Compare the revision in dark mode.

-   I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: (yes)

<img width="737" alt="Comparing revisions in dark mode." src="https://github.com/wagtail/wagtail/assets/1834167/8e5c034b-612e-477d-8cc1-2b1e3ccd2180">

### Technical details

-   Python version: 3.11
-   Django version: 3.2.29
-   Wagtail version: 5.0.1
-   Browser version: Chrome 113
-   Mac OS X: 12.6.1 Monterey with dark mode set to default

**Comment by lb-** [comment]
<img width="823" alt="Screenshot 2023-06-14 at 7 56 20 am" src="https://github.com/wagtail/wagtail/assets/1396140/296a49d0-cb8d-4017-b576-1cca800f7fd5">

Confirming this issue.

**Comment by thibaudcolas** [comment]
Thanks for the report @jessemenn! I thought this was on our list for dark theme tweaks but apparently not. Will work on a fix for the next 5.0 patch release.
